packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -736,10 +744,16 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 	/**
 	 * Submits a new signal to the server
 	 *
-	 * @param message - signal to submit
+	 * @param content - Content of the signal.
+	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	public submitSignal(message: IDocumentMessage): void {
-		this.emitMessages("submitSignal", [[message]]);
+	public submitSignal(content: unknown, targetClientId?: string): void {
+		const signal: ISentSignalMessage = {
```

Comment:
where does this get unwrapped? does the server do it? i worry the change is shape won't be handles by clients. over all this needs tests

Reply:
The server will process the `ISentSignalMessage`'s and send the signal to the specific client when specified.
The signals received by clients will look identical to how they are today - there is no breaking change / requirements for clients to all be on a new version.